packages/deadgods/src/utils/index.ts

```diff
@@ -153,7 +153,7 @@ async function fetchTokenAccounts(
   wallet: PublicKey,
   connection: Connection
 ): Promise<any> {
-  const resp = await window.xnft.connection.customSplTokenAccounts(wallet);
```

Comment:
`window.xnft.solana.connection` and `window.xnft.solana.publicKey` would probably be a better pattern to encourage? Wdyt?

Reply:
I think that's a good idea, would require finetuning the event we listen to because `connect` doesn't necessarily mean `window.xnft.solana.connection` exists. Will add those changes